From 84fefcadcde5d6c343db7f5c1744f6b1f070bc71 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Thu, 14 May 2009 11:30:40 -0400
Subject: [PATCH] Remove "inline" keyword from nl_object_priv().

"extern inline foo();" in a header file generates a compiler warning in
consumers of the library.  Also, it doesn't make any sense whatsoever.
For this reason, and the fact that this function is not used at all in
the file in which it's defined, I have removed the "inline" keyword.
---
 doc/Doxyfile.in          |    2 +-
 include/netlink/object.h |    2 +-
 lib/object.c             |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/doc/Doxyfile.in b/doc/Doxyfile.in
index 0c518b7..396fcbc 100644
--- a/doc/Doxyfile.in
+++ b/doc/Doxyfile.in
@@ -113,7 +113,7 @@ FULL_PATH_NAMES        = YES
 # If left blank the directory from which doxygen is run is used as the 
 # path to strip.
 
-STRIP_FROM_PATH        = 
+STRIP_FROM_PATH = /home/pjones/build/BUILD/libnl-1.1
 
 # The STRIP_FROM_INC_PATH tag can be used to strip a user-defined part of 
 # the path mentioned in the documentation of a class, which tells 
diff --git a/include/netlink/object.h b/include/netlink/object.h
index 751a1b3..241a2ad 100644
--- a/include/netlink/object.h
+++ b/include/netlink/object.h
@@ -55,7 +55,7 @@ extern int			nl_object_is_marked(struct nl_object *);
 /* Access Functions */
 extern int			nl_object_get_refcnt(struct nl_object *);
 extern struct nl_cache *	nl_object_get_cache(struct nl_object *);
-extern inline void *		nl_object_priv(struct nl_object *);
+extern void *			nl_object_priv(struct nl_object *);
 
 #ifdef __cplusplus
 }
diff --git a/lib/object.c b/lib/object.c
index 74f6e2d..845ca1d 100644
--- a/lib/object.c
+++ b/lib/object.c
@@ -376,7 +376,7 @@ struct nl_cache *nl_object_get_cache(struct nl_object *obj)
 	return obj->ce_cache;
 }
 
-inline void *nl_object_priv(struct nl_object *obj)
+void *nl_object_priv(struct nl_object *obj)
 {
 	return obj;
 }
-- 
1.6.2.2

